src/utils/modules.ts

```diff
@@ -1,7 +1,7 @@
 import {SyncRequestClient} from 'ts-sync-request/dist'
 import {Base64} from 'js-base64'
 
-import {UtilsVersions, JahiaModule} from '../global.type'
+import {JahiaModule} from '../global.type'
 
 export const getJahiaVersion = (version: string) => {
   if (version === 'UNKOWN') {
```

Comment:
just noticed the spelling, should be `UNKNOWN` everywhere, no?

Reply:
:( yes